web/templates/page/index.html.eex

```diff
@@ -1,4 +1,4 @@
-<h2>Pairs for <%= Timex.today %></h2>
+<h2>Pairs for <%= @year %> week <%= @week %></h2>
```

Comment:
how about:
`<h2>Pairs for week <%= @week %> of <%= @year %></h2>` ?

I think it sounds a bit less awkward.